code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -490,3 +490,10 @@
 
 /obj/item/clothing/head/winterhood/miner
 	icon_state = "winterhood_miner"
+
+/obj/item/clothing/suit/spookyghost
+	name = "spooky ghost"
+	desc = "this is obviously just a bedsheet, but maybe try it on?"
+	icon_state = "bedsheet"
+	user_vars_to_edit = list("name" = "Spooky Ghost", "real_name" = "Spooky Ghost" , "incorporeal_move" = 1, "appearance_flags" = 288) //288 is KEEP_TOGETHER|TILE_BOUND

```

Comment:
That would be because I had to work out the numbers on the fly to add it via VV because I was too lazy to recompile so I had the number 288 stuck in my head. I'll sort it in a commit tomorrow.